web/sites/default/settings.lando.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+$databases['default']['default'] = array (
```

Comment:
I know these are the lando defaults, but for flexibility, I think we should use this instead:
```
$lando_info = json_decode(getenv('LANDO_INFO'), TRUE);
  $databases['default']['default'] = [
    'driver' => 'mysql',
    'database' => $lando_info['database']['creds']['database'],
    'username' => $lando_info['database']['creds']['user'],
    'password' => $lando_info['database']['creds']['password'],
    'host' => $lando_info['database']['internal_connection']['host'],
    'port' => $lando_info['database']['internal_connection']['port'],
  ];
```
This will support changing credentials through Lando.